js/packages/components/settings/Accounts.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import { ScrollView, View, TouchableOpacity, Text } from 'react-native'
+import { useTranslation } from 'react-i18next'
+
+import beapi from '@berty-tech/api'
+import { useStyles } from '@berty-tech/styles'
+import { ScreenFC, useNavigation } from '@berty-tech/navigation'
+import {
+	useMessengerContext,
+	useThemeColor,
+	pbDateToNum,
+	closeAccountWithProgress,
+} from '@berty-tech/store'
+
+import { ButtonSettingV2, Section } from '../shared-components'
+import {
+	selectSelectedAccount,
+	setStateOnBoardingReady,
+} from '@berty-tech/redux/reducers/ui.reducer'
+import { useDispatch, useSelector } from 'react-redux'
```

Comment:
Move to the top of the file

Reply:
I think we have to code a tool that reorder imports 👍